Resources/Textures/SS220/Objects/Consumable/Food/shawarma.rsi/meta.json

```
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Added by gabova(discord)",
```

```suggestion
    "copyright": "Created by gabova(discord), modified by Whitly / Meower",
```
Крутые названия у коммитов, жаль что про копирайт забыли

---

Добавь модифицированный копирайт, указанный автором.

---

Напоминаю, что без упоминания автора/ссылки на оригинальный материал/ссылки на авторство спрайт не может использоваться, т.к. это нарушение лицензии.